CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Feature Development:
+Developing for cerner_splunk_ingredient should follow these guidelines for submitting issues and pull requests.
+
+### Submitting Issues
+Issues pertaining to cerner_splunk_ingredient should be submitted to Github's issue tracker.
+1. Issues will be triaged and marked with an appropriate label (e.g. bug, enhancement, question).
+  - A corresponding internal JIRA may be logged if there is a desire for internal discussion or code review.
+- The issue should be worked in a branch outside of the Cerner repo.
+- The issue will be marked for the next milestone following a pull request.
+
+### Pull Requests
+Pull requests for changes should be compared against the 'stable' branch.
+1. Your branch should use as few logical commits as possible with descriptive commit messages.
+  - _Note: these commits should **not** reference any issues, that will be done in the merge commit._
+- Your pull request should reference the issue it is related to.
+- The pull request is collectively reviewed and commented on, and must get two or more +1's, from reviewers and at least one maintainer of the repo.
+  - Any raised comments must be addressed and will be re-reviewed.
+- Once the pull request is approved, the changes will be merged by a maintainer.
+    - This is done manually, not by using the Github button, so that the fix version of metadata.rb can be incremented in the merge. ([Example](https://github.com/cerner/cerner_splunk/issues/41#issuecomment-70569000))
+    - The merge commit text should reference the issue and the pull request so that both are closed when the merge is pushed.
+
+# Release Process:
+1. A pull request with the release commit is made to the 'stable' branch.
```

Comment:
hm... we should update this text in the cerner_splunk one, because we no longer use a PR for releases since we increment version at merge of pull request time so release is just tagging. We use an issue proposing the tag sha, tag text, issue labeled release, and summarize changes.